### DISCUSSION TITLE
websites can also do something about their Co2 footprint

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#25] [Ideas]
Perhaps of interest for Dlite, helping reduce eco impact: https://beta.backspace.eco/

**Comment by renaudpawlak** [comment]
Thanks for the ref! I did not know it and will look closer at it when possible.

Side note: I am trying with DLite to enable software eco-design with Low-Code. That's a complicated challenge and I will probably not have short-term easy results... But I will focus as long as necessary or proven impossible ;)
